Dexterous-Rambutan/battle-code
# Feature server routes

#### What's this PR do?

Added user routes, to enable getUserById and addUser
#### What are the important parts of the code?

userController and routes.js
#### How should this be tested by the reviewer?

Run server/server.js and make post request to /api/users with req.body.user_handle.
This should return the newly created user object
Then make get request to /api/users/user_handle
This should return the user with user_handle: user_handle
Then make get request to /api/users/not_a_user
This should return null, since this user does not exist.
#### Is any other information necessary to understand this?
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

References #16 
#### Screenshots (if appropriate)
